admin/sql/CreateIndexes.sql

```diff
@@ -389,6 +389,7 @@ CREATE UNIQUE INDEX medium_format_idx_gid ON medium_format (gid);
 CREATE UNIQUE INDEX place_idx_gid ON place (gid);
 CREATE INDEX place_idx_name ON place (name);
 CREATE INDEX place_idx_area ON place (area);
+CREATE INDEX place_idx_geo ON place USING gist (ll_to_earth(coordinates[0], coordinates[1])) WHERE coordinates IS NOT NULL;
```

Comment:
This is the core addition. `ll_to_earth` is a function that essentially takes a long/lat position and computes a 3D point from it, assuming a perfectly spherical Earth. This 3D point is placed in a GiST index.